src/collect-install.rst

```diff
@@ -34,14 +34,21 @@ If you need a different version of Collect, you can download from the web and in
   
   You can also `download the ODK Collect APK <https://github.com/getodk/collect/releases/latest>`_ to your computer and load it on your device via `adb <https://developer.android.com/studio/command-line/adb.html>`_ or another tool like `AirDroid <https://www.howtogeek.com/105813/control-your-android-from-a-browser-with-airdroid/>`_.
 
-.. note::
-
-  On older Android devices (4.0 and earlier), ODK Collect required an external SD Card. This is no longer an issue because Android devices have internal storage. Virtually all current Android devices will run ODK Collect.
-
 .. tip::
 
   You can install ODK Collect on the `Android emulator <https://developer.android.com/studio/run/emulator>`_. This is especially helpful if you need to :doc:`project Collect onto a screen <projecting-collect>`.
 
+.. _collect-versions:
+
+Collect versions
+-----------------
+
+We generally recommend letting the Android Play Store automatically manage Collect updates. We strongly encourage anyone managing a data collection project to sign up for `the beta program <https://forum.getodk.org/c/releases/betas/19>`_ to read about upcoming changes and try their forms with any new version before it rolls out to all users. 
```

Comment:
"We generally recommend letting the Android Play Store automatically update Collect". So you don't have to say manage multiple times.

"before it is released to all users". Rolls out is a little colloquial.